.github/workflows/tests.yml

```
@@ -14,5 +14,5 @@ jobs:
 
     - uses: actions/checkout@v2
 
-    - name: Check our script for errors
-      run: shellcheck ./containers.sh ./gc.sh
+    - name: Check our scripts for errors
+      run: shellcheck ./docker_functions.sh ./create_containers.sh ./gc.sh
```

Is it possible to do this perhaps?

```suggestion
      run: shellcheck ./*.sh
```